tests/lib/rules/filename-naming-convention.posix.js

```diff
@@ -1271,3 +1271,95 @@ ruleTester.run(
     ],
   }
 );
+
+ruleTester.run(
+  "filename-naming-convention with option: [{ '**/components/featureA/*.*': '$1*' }]",
```

Comment:
Each ruleTester should always have the same options in this project. I noticed there are different options in this ruleTest.

Please let me know if you have any questions or concerns.